code/modules/emotes/emote_sounds.dm

```diff
@@ -0,0 +1,52 @@
+/*
+* Declare for storing species and gender specific emote sounds and make them overridable between species. 
+* Also the soundlist is loaded from a json array that sorts lists of sounds by emote name and optionally gender.
+*/
+
+//Mainly used for debugging and verifying everything loaded properly
+var/global/list/decl/emote_sounds/all_emote_decl = list()
+
+//merging proc for the merge assoc list proc
+/proc/merge_replace_if_not_null(var/a, var/b)
```

Comment:
IMO generic helpers like this should be in the helpers folder rather than mixed in with feature code.

Reply:
I agree with you on this. I just didn't know where to dump it. I couldn't even find where the other merge procs are supposed to be. So if you got suggestions let me know!